organization_invitation.go

```diff
@@ -2,16 +2,31 @@ package clerk
 
 import "encoding/json"
 
+type publicOrganizationData struct {
```

Comment:
:wrench: This type needs to be exported, similar to https://github.com/clerk/clerk-sdk-go/blob/000038eca0d2542ae5c318a332c82dc2514eaeb4/organization_membership.go#L16

Otherwise clients are going to have a hard time working with the data.